### PR TITLE
Update devDependencies

### PR DESCRIPTION
**Pull request by belochub**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,17 +39,17 @@
...
   "devDependencies": {
-    "@babel/cli": "^7.1.5",
-    "@babel/core": "^7.1.6",
-    "@babel/preset-env": "^7.1.6",
-    "eslint": "^5.9.0",
-    "eslint-config-metarhia": "^6.0.0",
-    "eslint-config-prettier": "^3.3.0",
-    "eslint-plugin-import": "^2.14.0",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
```
Comment (SemenchenkoVitaliy) [inline package.json]: The latest version of `@babel/core` is `7.3.3`: https://www.npmjs.com/package/@babel/core

File: package.json [hunk 2]
```diff
@@ -39,17 +39,17 @@
...
+    "@babel/preset-env": "^7.3.1",
+    "eslint": "^5.13.0",
```
Comment (SemenchenkoVitaliy) [inline package.json]: The latest version of `eslint` is `5.14.0`: https://www.npmjs.com/package/eslint

File: package.json [hunk 3]
```diff
@@ -39,17 +39,17 @@
...
+    "eslint-config-metarhia": "^7.0.0",
+    "eslint-config-prettier": "^4.0.0",
+    "eslint-plugin-import": "^2.16.0",
     "eslint-plugin-prettier": "^3.0.1",
     "metaschema": "0.0.28",
```
Comment (SemenchenkoVitaliy) [inline package.json]: The latest version of `metaschema` is `0.2.0`: https://www.npmjs.com/package/metaschema

Comment (belochub) [reply package.json]: It is not compatible with 0.0.28.

**Pull request merged**